docs/source/common/lightning_cli.rst

```diff
@@ -312,6 +312,45 @@ Similar to the callbacks, any arguments in :class:`~pytorch_lightning.trainer.tr
 :class:`~pytorch_lightning.core.datamodule.LightningDataModule` classes that have as type hint a class can be configured
 the same way using :code:`class_path` and :code:`init_args`.
 
+Alternatively, the user can provide the list of callbacks directly from the command line.
+
+.. code-block:: bash
+
+    $  python ... --trainer.callbacks=[{"class_path": pytorch_lightning.callbacks.EarlyStopping, "init_args": "patience": "5"}, ...].
+
+Lightning optionally simplifies the user command line where only the :class:`~pytorch_lightning.callbacks.Callback` name is required.
```

Comment:
```suggestion
Lightning optionally simplifies the user command line so that only the :class:`~pytorch_lightning.callbacks.Callback` name is required.
```

Reply:
Done !